src/electionguard/bignum.c

```diff
@@ -121,36 +121,32 @@ void pow_mod_p(mpz_t res, const mpz_t base, const mpz_t exp)
 {
     mpz_powm(res, base, exp, p);
 
-#ifdef DEBUG_PRINT
-    printf("Performing operation powmod (base^exp)%%p");
-    printf("\nbase = ");
+    TRACE_PRINT(("Performing operation powmod (base^exp)%%p"));
+    TRACE_PRINT(("\nbase = "));
     print_base16(base);
-    printf("\nexp = ");
+    TRACE_PRINT(("\nexp = "));
     print_base16(exp);
-    printf("\np = ");
+    TRACE_PRINT(("\np = "));
     print_base16(p);
-    printf("\nresult = ");
+    TRACE_PRINT((("\nresult = ")));
     print_base16(res);
-    printf("\n");
-#endif
+    TRACE_PRINT(("\n"));
 }
 
 void pow_mod_q(mpz_t res, const mpz_t base, const mpz_t exp)
 {
     mpz_powm(res, base, exp, q);
 
-#ifdef DEBUG_PRINT
-    printf("Performing operation powmod (base^exp)%%p");
-    printf("\nbase = ");
+    TRACE_PRINT(("Performing operation powmod (base^exp)%%p"));
```

Comment:
By using empty macros instead of compiler directives, we ensure that the api interactions are always valid no matter which configuration we build for